scripts/jekyll_build.sh.tpl

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+HOST="${HOST-localhost}"
+PORT="${PORT-12345}"
+RUNFILES=$(cd ${JAVA_RUNFILES-$0.runfiles}/%{workspace_name} && pwd -P)
+SOURCE_DIR="$RUNFILES/%{source_dir}"
+prod_dir="$RUNFILES/%{prod_dir}"
+bucket="%{bucket}"
+
+function serve() {
+  TDIR=$(mktemp -d)
+  RDIR=$(mktemp -d)
+  trap "rm -fr $RDIR $TDIR" EXIT
+  (cd $RDIR && \
+    jekyll serve --host "$HOST" --port "$PORT" -s "$SOURCE_DIR" -d "$TDIR")
+}
+
+function push() {
+  # Get gsutil
+  gs="${GSUTIL:-$(which gsutil 2>/dev/null || : )}"
+  if [ ! -x "${gs}" ]; then
```

Comment:
nit, optional: use `[[` instead of `[`.
`[` is a binary (`/usr/bin/[`) while `[[` is a builtin. AFAIK the former forks a new process, the latter doesn't.

Reply:
`[[` is a bashism... Indeed `[[` does not create a new process but it also fails with `/bin/sh` shebang